SSLError on startup

This was reported as an error for a cold start.

```
2019-05-16 02:07:44,023 [Controller] [ERROR] Ecobee Controller:get: Connection error for https://api.ecobee.com/1/thermostatSummary: HTTPSConnectionPool(host='api.ecobee.com', port=443): Max retries exceeded with url: /1/thermostatSummary?json=%7B%22selection%22%3A+%7B%22selectionMatch%22%3A+%22%22%2C+%22selectionType%22%3A+%22registered%22%2C+%22includesEquipmentStatus%22%3A+true%7D%7D (Caused by ConnectTimeoutError(<urllib3.connection.VerifiedHTTPSConnection object at 0x75bf1c50>, 'Connection to api.ecobee.com timed out. (connect timeout=60)'))
2019-05-16 02:07:44,027 [Controller] [ERROR] Thermostats instance wasn't dictionary. Skipping...
2019-05-16 13:01:59,996 [MainThread] [INFO ] Received Config from STDIN.
2019-05-16 13:02:00,009 [MainThread] [INFO ] UDI Polyglot v2 Interface 2.0.29 Starting...
2019-05-16 13:02:02,630 [Interface ] [INFO ] Connecting to MQTT... localhost:1883
2019-05-16 13:02:02,635 [Interface ] [ERROR] MQTT Connection error: An exception of type SSLError occurred. Arguments:
(1, 'getrandom() initialization failed. (_ssl.c:720)')
Traceback (most recent call last):
  File "/home/pi/.local/lib/python3.5/site-packages/polyinterface/polyinterface.py", line 325, in _startMqtt
    self._mqttc.loop_forever()
  File "/home/pi/.local/lib/python3.5/site-packages/paho/mqtt/client.py", line 1566, in loop_forever
    self.reconnect()
  File "/home/pi/.local/lib/python3.5/site-packages/paho/mqtt/client.py", line 994, in reconnect
    sock.do_handshake()
  File "/usr/lib/python3.5/ssl.py", line 996, in do_handshake
    self._sslobj.do_handshake()
  File "/usr/lib/python3.5/ssl.py", line 641, in do_handshake
    self._sslobj.do_handshake()
ssl.SSLError: getrandom() initialization failed. (_ssl.c:720)
2019-05-16 13:02:0
```

Just go this again:

```
2019-10-27 12:47:52,689 [MainThread] [INFO ] Controller:init: Initializing HueEmulator Controller 2.0.8
2019-10-27 12:47:52,713 [Interface ] [ERROR] Exception in thread Interface:
Traceback (most recent call last):
  File "/home/pi/.local/lib/python3.5/site-packages/polyinterface/polyinterface.py", line 334, in _startMqtt
    self._mqttc.loop_forever()
  File "/home/pi/.local/lib/python3.5/site-packages/paho/mqtt/client.py", line 1566, in loop_forever
    self.reconnect()
  File "/home/pi/.local/lib/python3.5/site-packages/paho/mqtt/client.py", line 994, in reconnect
    sock.do_handshake()
  File "/usr/lib/python3.5/ssl.py", line 996, in do_handshake
    self._sslobj.do_handshake()
  File "/usr/lib/python3.5/ssl.py", line 641, in do_handshake
    self._sslobj.do_handshake()
ssl.SSLError: getrandom() initialization failed. (_ssl.c:720)
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,7 @@
 # __Changelog for Polyglot Python Interface v2__
 
+### Version 2.0.34
+* Proper SSLError fix fix https://github.com/UniversalDevicesInc/polyglot-v2/issues/13
 
 ### Version 2.0.33
 * Added Added get_server_data and check_profile poly methods

--- a/polyinterface/__init__.py
+++ b/polyinterface/__init__.py
@@ -1,6 +1,6 @@
 from .polyinterface import Interface, Node, Controller, LOGGER, unload_interface, get_network_interface
 
-__version__ = '2.0.33'
+__version__ = '2.0.34'
 __description__ = 'UDI Polyglot v2 Interface'
 __url__ = 'https://github.com/UniversalDevicesInc/polyglot-v2-python-interface'
 __author__ = 'James Milne'

--- a/polyinterface/polyinterface.py
+++ b/polyinterface/polyinterface.py
@@ -358,9 +358,8 @@ class Interface(object):
                 self._mqttc.connect_async('{}'.format(self._server), int(self._port), 10)
                 self._mqttc.loop_forever()
                 done = True
-            except SSLError:
-                LOGGER.error("MQTT Connection SSLError: {}", exc_info=True)
-                LOGGER.error("Will retry in a few seconds")
+            except ssl.SSLError as e:
+                LOGGER.error("MQTT Connection SSLError: {}, Will retry in a few seconds.".format(e), exc_info=True)
                 time.sleep(3)
             except Exception as ex:
                 template = "An exception of type {0} occurred. Arguments:\n{1!r}"

